src/blob_serializer_deserializer-inl.h

```diff
@@ -0,0 +1,356 @@
+#ifndef SRC_BLOB_SERIALIZER_DESERIALIZER_INL_H_
+#define SRC_BLOB_SERIALIZER_DESERIALIZER_INL_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "blob_serializer_deserializer.h"
+
+#include <ostream>
+#include <sstream>
+#include <string>
+#include <type_traits>
+#include <utility>
+
+#include "debug_utils-inl.h"
+
+namespace node {
```

Comment:
A comment at the top of this that explains this is related to `node_snapshotable.cc` rather than `node_blob.h` would likely be helpful for future generations :-)

Reply:
@jasnell good point, done! Could you PTAL?